src/components/Users/UserListAndCard.tsx

```diff
@@ -49,10 +49,12 @@ export const UserStatusIndicator = ({
       ) : (
         <Badge
           variant="secondary"
-          className="bg-gray-100 whitespace-nowrap text-wrap"
+          className="bg-gray-100 inline-flex items-center gap-1 px-2 py-1 rounded-lg text-xs sm:text-sm text-gray-700 
+        max-w-full truncate overflow-hidden whitespace-nowrap"
         >
-          <span className="inline-block h-2 w-2 shrink-0 rounded-full bg-gray-500 mr-2" />
-          <span className="text-xs text-gray-700">{t("never_logged_in")}</span>
+          <span className=" h-2 w-2 shrink-0 rounded-full bg-gray-500 " />
```

Comment:
```suggestion
          <span className="h-2 w-2 shrink-0 rounded-full bg-gray-500" />
```
Lets not have spaces for no reason